gradle/libs.versions.toml

```diff
@@ -1,55 +1,42 @@
 [versions]
-managed-grpc = '1.48.1'
-managed-protobuf = '3.21.5'
+managed-grpc = '1.50.2'
+managed-protobuf = '3.21.8'
+managed-protobuf-gradle = '0.9.1'
 
-jackson-datatype-protobuf = '0.9.12'
+jackson-datatype-protobuf = '0.9.13'
 jaeger-core = '1.8.1'
 opentracing-grpc = '0.2.3'
 opentracing-mock = '0.33.0'
+micronaut-docs = '2.0.0'
 
 [libraries]
 
-boms-grpc = {module = "io.grpc:grpc-bom", version.ref = "managed-grpc"}
 managed-grpc-protobuf = { module = 'io.grpc:grpc-protobuf', version.ref = 'managed-grpc' }
 managed-grpc-stub = { module = 'io.grpc:grpc-stub', version.ref = 'managed-grpc' }
 managed-grpc-services = { module = 'io.grpc:grpc-services', version.ref = 'managed-grpc' }
+managed-protoc-grpc = { module = 'io.grpc:protoc-gen-grpc-java', version.ref = 'managed-grpc' }
 
-managed-protoc = { module = 'com.google.protobuf:protoc', version.ref = 'managed-protobuf' }
 managed-protobuf-java = { module = 'com.google.protobuf:protobuf-java', version.ref = 'managed-protobuf' }
-managed-protobuf-javalite = { module = 'com.google.protobuf:protobuf-javalite', version.ref = 'managed-protobuf' }
 managed-protobuf-java-util = { module = 'com.google.protobuf:protobuf-java-util', version.ref = 'managed-protobuf' }
-managed-protobuf-kotlin = { module = 'com.google.protobuf:protobuf-kotlin', version.ref = 'managed-protobuf' }
-managed-protobuf-kotlin-lite = { module = 'com.google.protobuf:protobuf-kotlin-lite', version.ref = 'managed-protobuf' }
-
-micronaut-discovery-client = { module = 'io.micronaut.discovery:micronaut-discovery-client' }
-micronaut-http = { module = 'io.micronaut:micronaut-http' }
-micronaut-http-netty = { module = 'io.micronaut:micronaut-http-netty' }
-micronaut-buffer-netty = { module = 'io.micronaut:micronaut-buffer-netty' }
-micronaut-http-server-netty = { module = 'io.micronaut:micronaut-http-server-netty' }
-micronaut-inject-java = { module = 'io.micronaut:micronaut-inject-java' }
-micronaut-management = { module = 'io.micronaut:micronaut-management' }
-micronaut-reactor-http-client = { module = 'io.micronaut.reactor:micronaut-reactor-http-client' }
-micronaut-runtime = { module = 'io.micronaut:micronaut-runtime' }
-micronaut-tracing = { module = 'io.micronaut:micronaut-tracing' }
-micronaut-validation = { module = 'io.micronaut:micronaut-validation' }
+managed-protoc = { module = 'com.google.protobuf:protoc', version.ref = 'managed-protobuf' }
 
-graal = { module = "org.graalvm.nativeimage:svm" }
+micronaut-docs-asciidoc-config-props = { module = 'io.micronaut.docs:micronaut-docs-asciidoc-config-props', version.ref = 'micronaut-docs' }
```

Comment:
Do we need this?  I thought it came with the internal build?

Reply:
resolved via https://github.com/micronaut-projects/micronaut-grpc/pull/606/commits/731368939b3ea41c349d4da8f6ef1ebf0d8f1a9a